libs/output-mapping/src/Writer/Helper/SliceCommandBuilder.php

```diff
@@ -39,7 +39,7 @@ public static function createProcess(
 
         return new Process(
             command: $command,
-            timeout: 7200.0,
+            timeout: 201600.0,
```

Comment:
Idealni kdybysme umeli tohle konfigurovat pres limity projektu, nebo alespon to mit variabilni podle velikosti backendu pro job 🤔 Mozna bych na to zalozil issue